### PR TITLE
PBI-13: create comment

### LINKED ISSUE #30: PBI-13: Teacher & Student Can comment and see the announcement in class
**Story Point: 13**

### PR DESCRIPTION
**Pull request by kanziebub**

### CODE REVIEW

**Review by nofamex — APPROVED** [review]
LGTM

File: src/Models/Comment.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ];
+
+    protected $hidden = [
+        'created_at',
+        'updated_at',
+    ];
+
+    public function announcement()
+    {
+        return $this->belongsTo(Course::class, 'announcement_id');
```
Comment (AldiNFitrah) [inline src/Models/Comment.php]: Announcement?

Comment (kanziebub) [reply src/Models/Comment.php]: *gasp

Comment (kanziebub) [reply src/Models/Comment.php]: eh what's wrong with that?

Comment (kanziebub) [reply src/Models/Comment.php]: done done

**Review by Zsaschz — APPROVED** [review]
i approb

**Review by nofamex — APPROVED** [review]
LGTM

**Review by IlmaMannix — APPROVED** [review]
i approb (2)

**Comment by kanziebub** [comment]
ikuzo 3 2 1 ...

**Pull request merged**